### ISSUE TITLE
Mouse cursor does not accept click in Remote Desktop

### ISSUE THREAD

**Issue by caiohaz** [FreeRDP/FreeRDP#2363]
When I use particular application in remote desktop some actions cause the mouse cursor no longer accept any click, I can move it but I can not click on any window, either within the Remote Desktop and even in my station, I can only use through the keyboard, and close the Remote Desktop window everything back to normal.

Command:
xfreerdp -u user -p password -d domain --app --plugin /usr/lib/x86_64-linux-gnu/freerdp/rail.so --data "%windir%\system32\RemoteApp\Myapp" -- server

My PC: Ubuntu 14.10 amd64 (3.16.0-28-generic)
Server: Windows 2008 R2

FreeRDP version 1.0.2

Thank Ever Since

**Comment by whande1992** [comment]
I'm having the same problem, when I use / app. If not put / app works correctly. But I need to solve the problem, does anyone know how to fix it?

**Comment by bmiklautz** [comment]
@whande1992 what version of FreeRDP did you test?

**Comment by bmullan** [comment]
I am encountering a very similar situation and was going to file a new issue but I'll put my notes here instead ... unless you want a new issue for my input.   

= = = = = = = = = = = 
Here are my notes:

FreeRDP Mouse click events for left key not working	

**Ubuntu 18.04 
freerdp-x11 installed out of repository**

$ **xfreerdp --version
This is FreeRDP version 1.1.0-beta1 (git n/a)**

NOTE: the "beta" designation for this version so knew I expect bugs but wanted to report this if it hadn't been already.

I used virt-manager to create a Windows 7 ultimate VM on  and have until recently been running windows remoteapp's on my Ubuntu Desktop but something had broken w the left mouse click in a remoteapp.

Running the Windows 7 VM and logging in using the virt-manager "console"
my Mouse can correctly left & right click as would be expected.

Then using the above xfreerdp version to try to access the Win7 ult VM at its ip address X.X.X.X
and run a RemoteApp (in my case I run %windir%/explorer

The command I execute on my ubuntu to start the remote app is:

**$ xfreerdp /cert-ignore /u:$USER_ID /p:$USER_PWD /drive:$LIN_WIN_DRIVE,$LINUX_SHARED_FOLDER +clipboard:on /sound:sys:alsa /app:'%windir%\explorer.exe' /v:$WIN_VM_IP**

Running the above xfreerdp command line does bring up windows explorer.exe in a window that pops up as a RemoteApp on my Ubuntu desktop.

- I can move the mouse fine
- _left mouse clicks are ignored,_ 
- right click does seem to work and brings up a menu

**So next step to id problem:**

If I just access the Win7 Ultimate VM without the RemoteApp using:
**xfreerdp /cert-ignore /u:$USER_ID /p:$USER_PWD /v:$WIN_VM_IP**

I get the full Win7 desktop and both left & right mouse clicks DO work correctly !

**Next step to id problem:**

I then removed xfreerdp-x11 and xfreerdp v1 from my machine and installed freerdp v2

Verifying it is now the version being used...

$ **xfreerdp --version**
**_This is FreeRDP version 2.0.0-dev (git n/a)_**

Using xfreerdp v2 and the same script to execute & start my Win7 RemoteApp:

$ **xfreerdp /cert-ignore /u:$USER_ID /p:$USER_PWD /drive:$LIN_WIN_DRIVE,$LINUX_SHARED_FOLDER +clipboard:on /sound:sys:alsa /app:'%windir%\explorer.exe' /v:$WIN_VM_IP**

Running the above xfreerdp command line brings up windows explorer.exe as a RemoteApp **_-and- both left & right mouse clicks do now work as expected!_**

So using freerdp v2 is a workaround in this use-case of mine but unless I remember wrong... the Apache Guacamole server only works with freerdp v1 as least from a thread by one of the guacamole developers:

> TL;DR - the FreeRDP API is a constantly-moving and undocumented
> target. There is an effort underway to add support for the 2.0 API,
> but it is difficult to convey just how enormous that task is. There is
> no simple solution. It will be done eventually.
> 
> The issue here is that there is essentially zero API compatibility
> between FreeRDP releases. With the number of API-breaking changes in
> 2.0.x, it's become clear that it is no longer possible to directly
> depend on FreeRDP without producing unmaintainably-conditional code.
> We need to implement an abstraction layer between Guacamole's RDP
> support and FreeRDP to isolate our codebase from this.
> 
> The issue tracking this 2.0.x support effort is:
> https://issues.apache.org/jira/browse/GUACAMOLE-249
> 
I don't know how many guacamole users implement rdp connections which may become impossible to use if this possible left-mouse click bug affects them.

Again, maybe this has been fixed already for v1 and I am not using some updated module.

Brian

**Comment by CharoteWu** [comment]
@akallabeth Sorry to bother you. I'm using freerdp 2.0.0-rc3 which has the problem described by @caiohasouza.And I just wonder that has the problem fixed by 2.0.0-rc4 since you closed the problem on 22 Nov 2018 just three days after the release of 2.0.0-rc4. If not, should I just try 2.4.0?

**Comment by akallabeth** [comment]
@CharoteWu Don´t remember, that was ages ago ;)
just update to something a little less ancient.
